content/en/docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -36,13 +36,21 @@ port: 20250
 serializeImagePulls: false
 evictionHard:
     memory.available:  "200Mi"
+    nodefs.available:  "10%"
+    imagefs.available: "15%"
+    nodefs.inodesFree: "5%"
 ```
 
-In the example, the Kubelet is configured to serve on IP address 192.168.0.8 and port 20250, pull images in parallel,
-and evict Pods when available memory drops below 200Mi. Since only one of the four evictionHard thresholds is configured,
-other evictionHard thresholds are reset to 0 from their built-in defaults.
-All other Kubelet configuration values are left at their built-in defaults, unless overridden
-by flags. Command line flags which target the same value as a config file will override that value.
+In this example, the Kubelet is configured with the following settings:
+
+1. address: The Kubelet will serve on IP address 192.168.0.8.
+2. port: The Kubelet will serve on port 20250.
+3. serializeImagePulls: Image pulls will be done in parallel.
+4. Eviction Thresholds: The Kubelet will evict Pods under the following conditions:
+   - When the node's available memory drops below 200Mi.
+   - When the node's main filesystem's available space is less than 10%.
+   - When the image filesystem's available space is less than 15%.
+   - When the node's main filesystem's free inodes are less than 5%.
```

Comment:
```suggestion
   - When more than 95% of the node's main filesystem's inodes are in use.
```
Let's assume readers can do the math here. If it makes people stop and think, I personally think that provides a benefit.